fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -501,6 +502,12 @@ public void execute() throws Exception {
                         PrepareStmt prepareStmt = prepareStmtContext.getStmt();
                         parsedStmt = prepareStmt.assignValues(executeStmt.getParamsExpr());
                         parsedStmt.setOrigStmt(originStmt);
+                        if (parsedStmt instanceof QueryStatement) {
+                            QueryStatement queryStmt = (QueryStatement) parsedStmt;
+                            queryStmt.setQueryType(QueryType.EXECUTE_QUERY);
+                            queryStmt.setParamsExpr(executeStmt.getParamsExpr());
+                            queryStmt.setExecuteStmtName(executeStmt.getStmtName());
+                        }
```

Comment:
we can use a new branch to resolve prepareStmt when prepareStmt has cache plan? don't need store value to query statement and handle it in `StatementPlanner.plan` ? and you can rollback to `StatementPlanner.plan` if meet anything error

Reply:
has create new PrepareStmtPlanner file to handle related logical